### PR TITLE
PR for issue 385 - Server script support to specify z/OS STC Jobname

### PR DESCRIPTION
**Pull request by awisniew90**
PR for issue 385

### CODE REVIEW

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -662,6 +662,14 @@ warning.variable.invalid=CWWKE0941W: The variable declaration {0} specified in s
 warning.variable.invalid.explanation=Only alphanumeric characters are allowed in variable names specified in the server.env file.
 warning.variable.invalid.useraction=Either modify the variable name to only use alphanumeric characters or move the definition of the variable to the bootstrap.properties file.
 
+error.zosProcStart.procedure.invalid=CWWKE0942E: The z/OS STC procedure {0} failed to start. The procedure name is invalid.
```
Comment (kaczyns) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: I am assuming that there will be an ID review of these messages as well.  I think 'invalid' is one of those words that does not translate well to other languages.

**Comment by awisniew90** [comment]
#build

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 2]
```diff
@@ -662,6 +662,14 @@ warning.variable.invalid=CWWKE0941W: The variable declaration {0} specified in s
...
+error.zosProcStart.procedure.invalid.explanation=The STC procedure specified by the WLP_ZOS_PROCEDURE property in server.env is invalid.
+error.zosProcStart.procedure.invalid.useraction=Verify that the procedure name is no longer than eight character.
+
+error.zosProcStart.jobname.invalid=CWWKE0943E: The z/OS STC procedure {0} failed to start. The job name, {1}, is invalid.
+error.zosProcStart.jobname.invalid.explanation=The STC job name specified by the WLP_ZOS_JOBNAME property in server.env is invalid.
+error.zosProcStart.jobname.invalid.useraction=Verify that the job name is no longer than eight characters.
```
Comment (chirp1) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Change "The z/OS STC procedure {0} failed to start. The job name, {1}, is invalid." to "The z/OS STC procedure {0} failed to start. The {1} job name is not valid. Ensure that the job name is no longer than eight characters and does not contain a comma. " <= I think that we can put the resolution to the problem in the message text as it is short enough.

Change "The STC job name specified by the WLP_ZOS_PROCEDURE property in server.env is invalid." to "The STC job name specified by the WLP_ZOS_PROCEDURE property in the server.env file is not valid. The procedure name must be no longer than 8 characters and must not contain a comma.

For the user action, change "Verify that the job name is no longer than eight characters." to "Verify that the job name is no longer than eight characters and that it does not contain a comma.

**Comment by chirp1** [comment]
Adam and I reviewed my updates, and agreed on a final tweak. My ID review is finished.

**Pull request merged**